pkg/cmd.go

```diff
@@ -54,3 +56,10 @@ func (a *Cmd) Delete(resources []*unstructured.Unstructured) error {
 	_, err := a.Deleter.Do()
 	return err
 }
+
+// Status returns the status given the input as a slice of unstructured resources
+func (a *Cmd) Status(resources []*unstructured.Unstructured) error {
```

Comment:
Maybe need to return the status result for this function. I didn't include the results as well for other commands in this package. It might not be a good choice. Can you add a TODO item here to think about what need to return from those functions? Thank you.

Reply:
added TODO